### PR TITLE
Fix styling issue on Islands page causing column text to be hard/impossible to read

### PR DESCRIPTION
**Pull request by Aaronm14**
### Description

One of the tables has a column with inline `style="color:white"` styling, which was fine in dark mode, but hard to read in light mode.  By default the site theme already adjusts the font color accordingly so this doesn't seem needed

### Screenshots

Before:
localhost-before

After (light mode):
localhost-after-light-mode

After (dark mode):
localhost-after-dark-mode

### CODE REVIEW

File: docs/reference/islands.md [hunk 1]
```diff
@@ -55,14 +55,14 @@ export default function InteractiveComponent({ defaultCount }) {
...
+<span>**clientOnly**</span> | `boolean` | `false` | When set to `true` the Island won’t be rendered on the server. This can be useful for components that rely on logic/libraries that can only run in the browser.
+<span>**hydrateOn**</span> | `"load" \| "visible" \| "idle"` | `"load"` | When rendering a page with Islands on the server, the output includes a script to initialize Islands on the client. Hydrating means downloading and initializing the Island component code, so using these different hydration types strategically to defer some of that work can help boost initial page load performance! See [Hydration Types](#hydration-types) below for more information.
+<span>**id**</span> | `string` | - | By default we generate a unique ID string for your island such as `island-123456`. But if you wish to provide your own island ID, you can use this prop.
+<span>**module***</span> | `React.Component` | - | Please keep in mind that the component passed in as the value of this prop must include the `?island` suffix as a part of the import RUL. Refer to the `InteractiveComponent` example at the top of the docs.
```
Comment (jontallboy) [inline docs/reference/islands.md:61]: This wasn't related to your changes here, but there is an extra `*` after `module` that we should remove.

**Review by jontallboy — CHANGES_REQUESTED** [review]
Looks good, just one comment that has nothing to do with your updates but we could cleanup since you are in here 😄

File: docs/reference/islands.md (hunk 1 above)
Comment (Aaronm14) [reply docs/reference/islands.md]: Good call - I removed in most recent commit.  I was wondering if maybe it was intentional to show like a footnote or something but I couldn't see any reason for it: https://github.hubspot.com/cms-react/reference/islands.html

Screenshot 2024-08-26 at 4 40 43 PM

**Review by jontallboy — APPROVED** [review]

File: docs/reference/islands.md (hunk 1 above)
Comment (jontallboy) [reply docs/reference/islands.md]: I think when I originally updated these docs I may have had a footnote for the module and never removed this asterisk after we decided to not dig into the details behind the module prop.

**Pull request merged**